AstronomyTests/MarsRoverClientTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MarsRoverClientTests.swift
+//  AstronomyTests
+//
+//  Created by Bradley Yin on 9/16/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import Astronomy
+
+class MarsRoverClientTests: XCTestCase {
+    func testFetchPhotos() {
```

Comment:
Maybe also include an assertion to affirm the count of the array is what you expect to know that you're getting the right data?